3. Installation/3. Docker Containers/systemd/README.md

```diff
@@ -4,22 +4,103 @@ Deploy [Rocket.Chat](https://github.com/RocketChat/Rocket.Chat) to Linux that ru
 
 ## Prerequisite
 
-You need to have [docker](https://docs.docker.com/linux/started/) and [docker-compose](http://docs.docker.com/compose/) installed.
+You need to have [docker](https://docs.docker.com/linux/started/) installed.
 
 ## How to run Rocket.Chat on systemd.
 
+1. Create a docker network `docker network create rocketchat_default`
+2. Create two service files: mongo.service and rocketchat.service
+3. Enable the service files
+4. Run the mongo.service
+5. Create the mongo-init-replica to enable opolog
+6. Run the rocketchat.service
+
+If you reboot the server, the services will start automatically in the right order.
+
+### The docker-container for the mongo-init-replica will remain in an exited state. Do NOT remove the container `mongo-init-replica`!
```

Comment:
@geekgonecrazy I did not delete my container because of some instruction like this: http://www.sohamkamani.com/blog/2016/06/30/docker-mongo-replica-set/

> In case we remove one of our containers by mistake, the data would also vanish. Using Docker volumes and setting the appropriate --dbpath when running mongod would prevent this from happening.

I am running this on CoreOS so I also safe my Volumes in Data Containers which are also in the exited state and should not be removed. So perhaps someone else knows the answer or I will try this in a test environment...